src/content/docs/en/guides/backend/xata.mdx

```diff
@@ -6,9 +6,142 @@ service: Xata
+
+### Environment configuration
+
+The quickest way to integrate is via the Xata CLI
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add -g @xata.io/cli@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn global install @xata.io/cli@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+The package will expose the `xata` command to your `$PATH` and now it's possible to initialize the integration:
+
+```shell
+xata init
+```
+
+:::note
+You can just execute the binary with `npx`, `pnpm dlx`, or `yarn dlx` if preferred.
+:::
+
+Once you follow the prompt, you should have a `.env` file in the root of your project with the following variables defined:
+
+```ini title=".env"
+XATA_API_KEY=hash_key
+
+# Xata branch that will be used
+# if there's not a xata branch with
+# the same name as your git branch
+XATA_FALLBACK_BRANCH=main
+```
+
+To have IntelliSense for your environment variables, edit or create the file `env.d.ts` in your `/src` directory to make your environment variables type-safe:
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly XATA_API_KEY: string;
+  readonly XATA_FALLBACK_BRANCH?: string;
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv;
+}
+```
+
+:::tip
+Read more about [environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
+:::
+
+And a `.xatarc` file with a `databaseURL`. Though it can be extended to have more definitions, such as the output path for your codegen, e.g.:
+
+```json title=".xatarc"
+{
+  "databaseURL": "https://{{ Xata DB URL }}",
+  "codegen": {
+    "output": "src/xata.codegen.ts"
+  }
+}
+```
+
+Additionally, the prompt may have assisted you in creating an instance of the SDK (Software Development Kit) and, if you chose TypeScript, you also have the types generated from your schema. Notice `@xata.io/client` was added to your `package.json` as well.
+
+Finally, your project should have the following structure:
+
+<FileTree title="Project Structure">
+- src/
+  - **xata.codegen.ts**
+  - **env.d.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+- **.xatarc**
+</FileTree>
+
+## Create your queries
+
+To illustrate how to make a query, let's query the first 50 posts from Xata's Sample Blog Database.
+
+```astro title="src/pages/blog/index.astro"
+---
+import { getXataClient } from "../xata.codegen";
+
+const xata = getXataClient();
+const { records } = await xata.db.Posts.getPaginated({
+  pagination: {
+    size: 50
+  }
+})
+---
+
+<ul>
+  {records.map(post) => (
+    <li>{post.title}</li>
+  )}
+</ul>
+```
+
+Since the SDK is generated dynamically, and will be updated any time you tweak your schema, it's better to leave it untouched. In case you wish to hoist a few common queries to keep your code base DRY, it's a best practice to create an additional file where your queries can be re-used.
```

Comment:
```suggestion
For common queries that you may wish to reuse, you can also create an additional file.
```
I'm not sure that was as helpful as getting to the instructive point sooner.  :smile: It's not as important to explain why, as to model the code that will help the reader be successful.  Can you add an example of a file here, including title, that holds a reusable query? I think that would be a great addition! Then, show a `.astro` file that uses one of these queries.

Reply:
My point was just to say "don't make changes to the generated code because once schema updates, your changes will be overwritten".

So how about I say _that_ instead of what I wrote initially?? 😅